test/rules/best-practises/no-empty-blocks.js

```diff
@@ -1,27 +1,75 @@
-const { assertNoWarnings, assertErrorMessage, assertWarnsCount } = require('../../common/asserts')
+const {
+  assertNoWarnings,
+  assertErrorMessage,
+  assertWarnsCount,
+  assertErrorCount,
+} = require('../../common/asserts')
 const linter = require('../../../lib/index')
 const { contractWith, funcWith } = require('../../common/contract-builder')
 
 describe('Linter - no-empty-blocks', () => {
-  const EMPTY_BLOCKS = [
-    funcWith('if (a < b) {  }'),
-    contractWith('struct Abc {  }'),
-    contractWith('enum Abc {  }'),
-    contractWith('constructor () {  }'),
-    'contract A { }',
-    funcWith('assembly {  }'),
-  ]
+  describe('Basic empty blocks', () => {
+    const EMPTY_BLOCKS = [
+      funcWith('if (a < b) {  }'),
+      funcWith('for(uint i = 0; i < 10; i++) {  }'),
+      funcWith('while(true) {  }'),
+      funcWith('do {  } while(true);'),
+      funcWith('try foo() {  } catch {  }'),
+      contractWith('struct Abc {  }'),
+      contractWith('enum Abc {  }'),
+      contractWith('constructor () {  }'),
+      'contract A { }',
+      funcWith('assembly {  }'),
+    ]
 
-  EMPTY_BLOCKS.forEach((curData) =>
-    it(`should raise warn for empty blocks ${label(curData)}`, () => {
-      const report = linter.processStr(curData, {
-        rules: { 'no-empty-blocks': 'warn' },
+    EMPTY_BLOCKS.forEach((curData) =>
+      it(`should raise warn for empty block: ${label(curData)}`, () => {
+        const report = linter.processStr(curData, {
+          rules: { 'no-empty-blocks': 'warn' },
+        })
+        assertWarnsCount(report, curData.includes('try') ? 2 : 1)
```

Comment:
I'd rather define a separate testcase for the `try` statement since it's the only on with two empty blocks, and would like to keep test logic as simple as possible